msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md

```diff
@@ -159,7 +177,7 @@ if (uriList.length > 0) {
 }
 ```
 
-**Calling viewImages API by URL**
+**Calling `viewImages` API by URL**:
 
 ```javascript
 View Images by URL:
```

Comment:
Could you please add // in front of first line in this snippet? This is a comment.

Reply:
@Wajeed-msft , please approve the PR